apps/mull-ui/src/app/pages/create-event/date-calendar/date-calendar.scss

```
@@ -0,0 +1,27 @@
+@import '../../../../variables.scss';
+
+.date-calendar-container {
+  & .error-message {
+    text-align: center;
+  }
+}
+
+.date-calendar {
+  display: flex;
+  justify-content: center;
```

Like @JimLau49  mentioned many times, using flex might not always be necessary, you can center this by using margin auto.

```suggestion
  margin: auto;
```